### DISCUSSION TITLE
Welcome to Graphormer Discussions!

### DISCUSSION THREAD

**Discussion by zhengsx** [microsoft/Graphormer#105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ankit7699** [comment]
Is the code provided in the repository built entirely from scratch, or does it include a pre-trained model available in this GitHub repository?

**Comment by alisuxSZU** [comment]
我想做一个简单的应用，想了解一下训练完成后怎么导出部署训练好的模型，对fairseq不是很熟。

**Comment by HaruteGx** [comment]
First, this is an excellent piece of work, which is why I wanted to try it. However, as of October 2025, the SAS for this work has expired. Would it be possible to renew or update the SAS? If there are any commercial or licensing terms involved, please let me know.
首先这是一份很优秀的工作，然而如今是2025.10月，该工作的sas已经过期了，是否可以继续更新SAS（若尤其他商业条件或许可条件请告知）
